test/remote/gateways/remote_credorax_test.rb

```diff
@@ -64,11 +64,13 @@ def test_successful_purchase_with_extra_options
     assert_equal 'Succeeded', response.message
   end
 
+  # Having processor-specification enabled in Credorax test account causes 3DS tests to fail without an r1 parameter.
```

Comment:
Consider moving these comments down to just above the `processor` line to help draw the line between `r1` and `processor`.